common.go

```diff
@@ -16,33 +16,58 @@ import (
 // Logger used to log any output
 type Logger = log.Logger
 
+const (
+	MYSQL_DRIVER    = "mysql"
+	SQLITE_DRIVER   = "sqlite3"
+	POSTGRES_DRIVER = "postgres"
+)
+
+var (
+	errNoDriverName = errors.New("No driver name")
+	errEmptyConnURL = errors.New("URL cannot be empty")
+	errInvalidDSN   = errors.New("Invalid DSN")
+)
```

Comment:
Can you write all errors in lower case?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Updated.